js/tests/unit/alert.spec.js

```diff
@@ -207,4 +207,26 @@ describe('Alert', () => {
       expect(Alert.getInstance(div)).toEqual(null)
     })
   })
+
+  describe('getOrCreateInstance', () => {
+    it('should return alert instance', () => {
+      fixtureEl.innerHTML = '<div></div>'
+
+      const div = fixtureEl.querySelector('div')
+      const alert = new Alert(div)
+
+      expect(Alert.getOrCreateInstance(div)).toEqual(alert)
+      expect(Alert.getInstance(div)).toEqual(Alert.getOrCreateInstance(div, {}))
```

Comment:
Isn't the second expect kind of redundant? If `getOrCreateInstance` equals `alert` already does it makes sense to test `getInstance` separately as this is used by `getOrCreateInstance`?

Reply:
I want to show this usage, but can omitted too